docs/index.rst

```diff
@@ -10,6 +10,14 @@ parallelize, Just-In-Time compile to GPU/TPU, and more.
    Please see `Distributed arrays and automatic parallelization <https://jax.readthedocs.io/en/latest/notebooks/Distributed_arrays_and_automatic_parallelization.html>`_ tutorial and the :ref:`jax-array-migration`
    guide for more information.
 
+
+User Specific Guides
```

Comment:
Using a heading here makes the toctrees below this appear to be part of the heading.

Also, it's not clear how the "Beginner" link here is differentiated from "Getting Started" just below.

Reply:
Theres two levels of discussion here.

The first is whether we want to move these or not. The reason I had left them is because they were the most visited links off of the home page.

If we want to move them I propose Installing, Quickstart, and how to think go in the Beginner guide, and sharp bits go into advanced.

Happy to go either which way